### PR TITLE
Legato Service Order - OpenAPI 3.0 Files

### PR DESCRIPTION
**Pull request by jtpugac**

### CODE REVIEW

File: api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json [hunk 1]
```diff
@@ -0,0 +1,1391 @@
+{
+  "openapi" : "3.0.0",
+  "info" : {
+    "description" : " **Legato Service Order Management**\n\nThe intent of this API is to provide a consistent and standardized mechanism to manipulate the Service instances (and in turn the Service fulfillment) through Service Orders. The Service Order API can be invoked by the Business Applications (BA) systems to initiate Service Orders that manipulate the Service fulfillment process by the Service Orchestration Function (SOF).\n\nThe scope of this API introduces the following capabilities to the API:\n\n•\tADD: Request a new Service\n\n•\tMODIFY: Change an existing Service\n\n•\tDELETE:  Delete an existing Service\n\n•\tThe client could also retrieve a single, or set, of Service Orders and Service Order Items based upon a specified filter.",
+    "version" : "4.0.1_inProgress-oas3",
```
Comment (karthik-sethuraman) [inline api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json]: As per Git process, we should probably label this version as MEF_W99_v0.2.0

File: api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json [hunk 2]
```diff
@@ -0,0 +1,1391 @@
+{
+  "openapi" : "3.0.0",
+  "info" : {
+    "description" : " **Legato Service Order Management**\n\nThe intent of this API is to provide a consistent and standardized mechanism to manipulate the Service instances (and in turn the Service fulfillment) through Service Orders. The Service Order API can be invoked by the Business Applications (BA) systems to initiate Service Orders that manipulate the Service fulfillment process by the Service Orchestration Function (SOF).\n\nThe scope of this API introduces the following capabilities to the API:\n\n•\tADD: Request a new Service\n\n•\tMODIFY: Change an existing Service\n\n•\tDELETE:  Delete an existing Service\n\n•\tThe client could also retrieve a single, or set, of Service Orders and Service Order Items based upon a specified filter.",
```
Comment (karthik-sethuraman) [inline api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json]: As per OpenAPI 3.0 spec, descriptions should be in the CommonMark dialect of Markdown:  https://commonmark.org/help/ - basically we would need to replace \n with \ and start a new line, * for list, etc

File: api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json [hunk 3]
```diff
@@ -0,0 +1,1391 @@
+{
+  "openapi" : "3.0.0",
```
Comment (karthik-sethuraman) [inline api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json]: lets use OpenApi 3.0.2  as it fixes some bugs

File: api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json [hunk 4]
```diff
@@ -0,0 +1,1391 @@
...
+  "info" : {
+    "description" : " **Legato Service Order Management**\n\nThe intent of this API is to provide a consistent and standardized mechanism to manipulate the Service instances (and in turn the Service fulfillment) through Service Orders. The Service Order API can be invoked by the Business Applications (BA) systems to initiate Service Orders that manipulate the Service fulfillment process by the Service Orchestration Function (SOF).\n\nThe scope of this API introduces the following capabilities to the API:\n\n•\tADD: Request a new Service\n\n•\tMODIFY: Change an existing Service\n\n•\tDELETE:  Delete an existing Service\n\n•\tThe client could also retrieve a single, or set, of Service Orders and Service Order Items based upon a specified filter.",
+    "version" : "4.0.1_inProgress-oas3",
+    "title" : "Legato API Service Order",
+    "x-logo" : {
```
Comment (karthik-sethuraman) [inline api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json]: We can get rid of the x-logo tag

File: api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json (hunk 1 above)
Comment (jtpugac) [reply api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json]: FIxed with recommendation.

File: api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json (hunk 4 above)
Comment (jtpugac) [reply api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json]: Fixed. Remove x-logo tag.

File: api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json (hunk 3 above)
Comment (jtpugac) [reply api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json]: Changed to 3.0.2 in YAML. I assume that is the only change needed.

File: api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json (hunk 2 above)
Comment (jtpugac) [reply api/service-order/LegatoServiceOrder-4.0.1_10022019-oas3-swagger.json]: Using </p>.  Will that work ?

**Pull request merged**